main.html

```diff
@@ -0,0 +1,107 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"/>
+</head>
+<body>
+    <!-- home section start -->
+    <div class="home">
+        <h1>Communet</h1>
+    </div>
+    <!--about us page-->
+
+    <section class="about-us">
+        <div class="ab-container">
+            <div class="row">
+                <div class="col-sm-5">
+                    <h1 class="text-center about-us-h">About us</h1>
+                    <p class="about-us-p">Our idea is that you create a communet profile, users can apply to join, you reach economic agreements with them (shopping, chores, food, utilities, hobbies) and everyone agrees beforehand, and the system manages the expenses, reminds users of tasks, keeps track of who needs to do what, etc. (Ofc the money management part would be a mock up)
+                    </p>
+                </div>
+                <div class="col-sm-5">
+                    <div class="img-class" alt="">
+                        <img src="images/cm2.png" alt="">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+
+    <!--our team page-->
+    <div class="container">
+        <div class="header">
+            <h1>Our Team</h1>
+        </div>
+        <div class="sub-container">
+            <div class="teams">
+                <img src="images/m2.svg" alt="">
+                <div class="name">Faez Zangariya</div>
+                <div class="desig">Developer</div>
+                <div class="about">Student third year works with Flake8+Documentation</div>
+                <div class="social-links">
+                    <a href="https://github.com/FaezZangariya"><i class="fa-brands fa-github"></i></a> <!--for the links added only for the github..-->
+                    <a href="#"><i class="fa-brands fa-discord"></i></a>
+                    <a href="#"><i class="fa-brands fa-slack"></i></a>
+                </div>
+            </div>
+            <div class="teams">
+                <img src="images/m1.svg" alt="">
+                <div class="name">Ron Turetzky</div>
+                <div class="desig">Developer</div>
+                <div class="about">Bioinformatics DevOps, Blockchain enthusiast, professional task management software implementer/freak, activist , and avid nerd.</div>
+                <div class="social-links">
+                    <a href="https://github.com/ronturetzky"><i class="fa-brands fa-github"></i></a>
+                    <a href="#"><i class="fa-brands fa-discord"></i></a>
+                    <a href="#"><i class="fa-brands fa-slack"></i></a>
+                </div>
+            </div>
+            <div class="teams">
+                <img src="images/m2.svg" alt="">
+                <div class="name">Elias Assy</div>
+                <div class="desig">Developer</div>
+                <div class="about">Student third year works with webpage</div>
```

Comment:
same ^^ssssssssssssssssssssssssssssssssssssssss
```suggestion
                <div class="about">Student third year <general description></div>
```